europe/README.EUROPE

```diff
@@ -49,6 +49,30 @@ examples of use.
 * nkgrf03vel_realigned_xy.ct2
 * nkgrf03vel_realigned_z.gtx
 
+### Ireland: WGS84/ETRS89 ellipsoidal heights -> OSGM15
+
+*Source*: [Ordnance Survey](https://www.ordnancesurvey.co.uk/docs/gps/OSTN15-OSGM15_DevelopersPack.zip)
```

Comment:
It would be better to link to this site: https://www.ordnancesurvey.co.uk/business-and-government/help-and-support/navigation-technology/os-net/formats-for-developers.html

It offers more information for those who may need it while also listing a link to the original grids.

This also applies below for Northern Ireland